dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js

```diff
@@ -20,6 +20,10 @@ const interceptPlayEvent = (id) => {
 		},
 	);
 };
+
+const getOverlaySelector = (videoId) =>
+	`[data-cy="youtube-overlay-${videoId}"]`;
+
```

Comment:
Nitpick (so I'm fine if you don't want to do this) but could this be inlined?

Reply:
Sure will do